development-vm/replication/sync-aws-elasticsearch.sh

```diff
@@ -2,111 +2,119 @@
 
 set -eu
 
-USAGE_LINE="$0 [options] [index_name ...]"
-USAGE_DESCRIPTION="Download Elasticsearch index archives and import into another instance.
+export USAGE_LINE="$0 [options] [index_name ...]"
+export USAGE_DESCRIPTION="Download Elasticsearch index archives and import into another instance.
```

Comment:
Do these need to be exported?

Reply:
Looks like I misinterpreted a shellcheck error.  The exports are not needed.